src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
If all the other environment variable (e.g. `ROS_ROOT`, `ROS_MASTER_URI`, `ROS_DISTRO`, the completion scripts, and every custom env var) need to be handled explicitly (implying patches on the Debian side) why should the RPP be any different? Is it really worth avoiding a simple patch which appends the prefix as proposed in this PR for `rospkg` as well as `rospack`?